Client/Experiments/Experiments.swift

```diff
@@ -107,6 +107,9 @@ enum Experiments {
         let profilePath: String?
         if AppConstants.isRunningUITests || AppConstants.isRunningPerfTests {
             profilePath = (UIApplication.shared.delegate as? UITestAppDelegate)?.dirForTestProfile
+        } else if AppConstants.isRunningUnitTest {
+            let dir = URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
+            profilePath = dir.path
```

Comment:
For posterity's sake, this fixes an issue in Bitrise builds where unit tests saw this error:
```
2022-12-01 15:39:34.744 [Error] [Experiments.swift:166] Experiments > Nimbus didn't get to create, because of a nil dbPath
```
Which would cause any jexl queries or any interactions with Nimbus to be untestable since Nimbus was instrumented as a noop due to the error. As far as I'm aware, all this does is generate a temporary directory for the test run, which we then use to instantiate the Nimbus DB.